cirq/protocols/approximate_equality_test.py

```diff
@@ -38,6 +40,9 @@ def test_approx_eq_mixed_primitives():
     assert not cirq.approx_eq(1, 1.0 + 1e-10, atol=1e-11)
 
 
+@pytest.mark.skipif(sys.platform.startswith('win'),
```

Comment:
Let's skip if `float128` or `complex256` is not supported like https://github.com/quantumlib/Cirq/blob/0452fd817fd6a52e23682712484a8a5c047d30d0/cirq/google/sim/xmon_simulator_test.py#L375

Reply:
Adjusted skip conditions in both adjusted tests.